### DISCUSSION TITLE
Error The Serverless Function

### DISCUSSION THREAD

**Discussion by JacquelineCasali** [vercel/community#2065] [Help] [answered]
### Problem Description

The Serverless Function "front/wsgi.py" is 64.74mb which exceeds the maximum size limit of 50mb.
in Vercel
I'm using django/python

how to solve the problem

**Comment by afjol77** [comment]
Same problem here. Please let me know if you fixed it

**Reply by JacquelineCasali** [reply]
No
Do you have any idea what it could be?

**Reply by afjol77** [reply]
No. I have shifted to Replit platform.

**Comment by amyegan** [comment, marked as answer]
The [Serverless Function Size](https://vercel.com/docs/concepts/limits/overview#serverless-function-size) is 50MB. There are many things that can cause the size to become too large, but dependencies seem to be the most common. We have another open discussion where you can find various causes and solutions: https://github.com/vercel/community/discussions/103
